### ISSUE TITLE
Ability to assign points to a survey question

### ISSUE THREAD

**Issue by gplumley** [Esri/Survey123Community#219]
Our office is wanting to assign points to each question within the survey and be able to calculate the totals based on the inputs. Currently I've just added the points to the questions as a reference for staff's needs. 
Sample question:

![capture](https://cloud.githubusercontent.com/assets/15220579/10620187/2a861966-7726-11e5-8bb2-00324e56bf44.PNG)

**Comment by RKnepper3** [comment]
I'd be interested in this as well, I was just thinking about something like this but for cost estimating.  It would be nice to assign a value to answers.  I know you could create some long complicated logic statement to end up with something like this but I would love to see an easy way,

**Comment by survey123** [comment]
Will this help?

![untitled](https://cloud.githubusercontent.com/assets/10968332/10645876/d720cd1a-77e4-11e5-9857-cb32e3659551.png)

The trick would be to properly set the 'name' values in the choice list to integers. I used the default *rating* choice list included in our template. Note that to avoid any issues I cast the responses to integers using int(${questioname}).

It may also be handy for you to set the esriFieldTypeInteger value in the bind::esri:fieldType column to ensure that both the Calculate and other questions are stored as integers in your feature service.

**Comment by gplumley** [comment]
Yes, that helps to see how we can calculate the fields. I’m still trying to figure out how it can be done in our case.  I'm trying to see how this can be done within our survey since it isn't as simple as your sample was.
